src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/ChainPal.cs

```diff
@@ -49,7 +41,7 @@ internal sealed partial class ChainPal : IDisposable, IChainPal
             return status.dwError == 0;
         }
 
-        public IEnumerable<X509ChainElement> ChainElements
+        public X509ChainElement[] ChainElements
```

Comment:
It's amazing how different that sentence reads when you see comma/comma instead of comma/period.

For me the properties/methods debate for this is in the middle grey area.  I like the symmetry that the author introduced, but I don't like allocating properties.  So if someone wanted to change the properties to methods as a change in and of itself, I'd likely support it; but as part of another change I'd say it was excessively noisy.  Probably.